.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Special notes for your reviewer**:
+
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
+
+**Release note**:
+<!--  Write your release note:
+1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
+2. If no release note is required, just write "NONE".
```

Comment:
We don't have tooling to scrape these.  Changes that require user changes should instead be documented in a Bugzilla linked from the PR subject with a `Bug ###` prefix.

Reply:
But I can see some people also opening issues in Github but not only Bugzilla, can we use https://github.com/github-changelog-generator/github-changelog-generator to generate kind of CHANGELOG?